INSTALL.md

```diff
@@ -6,7 +6,7 @@ Qiita is pip installable, but depends on some non-python packages that must be i
 Install the non-python dependencies
 -----------------------------------
 
-* [PostgreSQL](http://www.postgresql.org/download/) (we have tested most extensively with 9.3)
+* [PostgreSQL](http://www.postgresql.org/download/) (minimum required version 9.3.0, we have tested most extensively with 9.3)
```

Comment:
What's the motivation for this? Also, we should make a note in the ChangeLog.md and update `qiita-test-install`.